app/src/vuex/modules/session.js

```diff
@@ -172,6 +172,9 @@ export default ({ apollo }) => {
       const allSessionAddresses = await dispatch("getAllSessionAddresses")
       commit("setAllSessionAddresses", allSessionAddresses)
 
+      // update registered topics for emails
+      dispatch("updateEmailRegistrations")
```

Comment:
We could check here if the user is signed in or not with rootState and thus prevent throwing the error of `Notifications need a signed in user` for no reason

Reply:
```
      rootState: {
        account: { userSignedIn },
      },
```
Althought I think we need to include a `setTimeOut` because it takes a bit for the Firebase observer to change this

Reply:
I had to add the Timeout here for example, otherwise it would redirect to the Paywall: https://github.com/luniehq/lunie/pull/4495/commits/80938ca978bd93943a2c889c89d6d838373f7f42

Reply:
https://github.com/luniehq/lunie/commit/80938ca978bd93943a2c889c89d6d838373f7f42#r40626643

Reply:
Should we do this? Feels strange to get that error also if the user is simply not logged in

Reply:
o.O that is not the case, is it?

Reply:
ahh I get it